### DISCUSSION TITLE
NSA test by using B210

### DISCUSSION THREAD

**Discussion by Sivanesh1992** [srsran/srsRAN_4G#1167] [Q&A] [answered]
Dear All,

Any one tested 5G NSA by using usrp B210.
I am using below link to test NSA by using one UsrpB210

https://docs.srsran.com/projects/4g/en/latest/app_notes/source/5g_nsa_cots/source/index.html

but LTE only attaching ,not able to see the 5G network in NSA mode.

I want to know we need two usrpB210 to test NSA 5G ?

If Yes please guide me how to Deploy 5G NSA by using two usrpB210.

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @Sivanesh1992, 

You cannot use a B210 for 5G NSA networks. This is due to the fact that srsRAN 4G requires two independent RF chains in the USRP to create both the 4G and 5G bearers required for 5G NSA. We recommend instead using an X310 and N310 USRP. 

It is possible to create a 5G NSA set-up using two B210s, although this is not something we officially support. Doing so would require a good knowledge of UHD, srsRAN, and changes to the srsRAN source-code, as well as external clock sources for syncing the USRPs. Better to just use one of the supported USRPs instead.
